### ISSUE TITLE
Where is jquery.mmenu.scss?

### ISSUE THREAD

**Issue by Toutouwai** [FrDH/jQuery.mmenu#551]
The SCSS for the default stylesheet seems to be missing.
The src folder contains `jquery.mmenu.oncanvas.scss` but not `jquery.mmenu.scss`.

**Comment by FrDH** [comment]
It's a combination of jquery.mmenu.oncanvas.scss and jquery.mmenu.offcanvas.scss

**Comment by Toutouwai** [comment]
The organisation of the SCSS files is not intuitive. The default behaviour of jQuery.mmenu is off-canvas yet jquery.mmenu.offcanvas.scss is in the "addons" folder?

And if there is no jquery.mmenu.scss then how does one follow the guidelines [here](http://mmenu.frebsite.nl/documentation/custom-css/concatenate-files.html)?
> /* custom-mmenu.css */
@import "path/to/jquery.mmenu";
@import "path/to/addons/jquery.mmenu.navbars";
@import "path/to/addons/jquery.mmenu.searchfield";

**Comment by FrDH** [comment]
The documentation needs to be updated, sorry about that.
FYI: there used to be a file `jquery.mmenu.scss` that included the two seporate files, now that's done by gulp.

**Comment by Jubke** [comment]
I think the change of the file structure in 5.6
> '21. Changed file structure and directory names.

might actually have been an API breaking change for many users that import scss files manually like above. It should have happened in a major version change I think. Next time maybe ;)

**Comment by FrDH** [comment]
Yeah sorry about that :/

@Toutouwai I did my best to explain the [on- and off-canvas behavior](http://mmenu.frebsite.nl/on-and-off-canvas.html).

**Comment by Akhoy** [comment]
@FrDH I want an off-canvas menu. I've imported all the the oncanvas scss files along with offcanvas.scss files. But what I've noticed is even though, I give offCanvas as the following in the js file, it still opens from the left side. 

`offCanvas:{
         	position: "right",
         	zposition:"front"
         }`

Anything more to be done here? This only happened after I manually referenced the scss files. When I tried with all.min.css earlier, it worked fine.
